vq_occupancy_compressor_node:
  ros__parameters:
    input_file: "/home/ryo/tsudanuma/maps/tsudanuma-challenge-all.ply"
    voxel_size: 0.1
    block_size: 13
    min_points_threshold: 2

    
    publish_occupied_voxel_markers: true
    save_hdf5: true
    hdf5_output_file: "/home/ryo/raspicat-sim-docker-vq/raspicat_vq_ws/src/maps/tsudanuma_voxelsize_01_compressed_map.h5"
    
    save_raw_hdf5: true
    raw_hdf5_output_file: "/home/ryo/raspicat-sim-docker-vq/raspicat_vq_ws/src/maps/tsudanuma_voxelsize_01_raw_map.h5"
